x-pack/plugins/secops/public/components/drag_and_drop/helpers.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DropResult } from 'react-beautiful-dnd';
+import { Dispatch } from 'redux';
+import { ActionCreator } from 'typescript-fsa';
+
+import { timelineActions } from '../../store';
+import { dragAndDropActions } from '../../store/local/drag_and_drop';
+import { IdToDataProvider } from '../../store/local/drag_and_drop/model';
+import { DataProvider } from '../timeline/data_providers/data_provider'; // tslint:disable-line:no-unused-variable (linter bug!)
```

Comment:
Same comment as previous. We should just remove the deprecated linter option and this code to not pollute things.